lib/libgcc_eh/Makefile.inc

```diff
@@ -1,11 +1,12 @@
 # $FreeBSD$
 
 COMPILERRTDIR=	${SRCTOP}/contrib/compiler-rt
-.if defined(LIBCHERI)
-# XXXAR: we should do this for MIPS too
+.if ${MK_CHERI} != "no"
+WANT_CHERI?=	hybrid
 UNWINDINCDIR=   ${SRCTOP}/contrib/cheri-libunwind/include
 UNWINDSRCDIR=   ${SRCTOP}/contrib/cheri-libunwind/src
 .else
+WANT_CHERI=hybrid
```

Comment:
This line doesn't make sense to me.  Why ask for hybrid when we're not supporting cheri?